### PR TITLE
JAMES-2997 Drop Cassandra schema version prior version 5

### PR DESCRIPTION
**Pull request by chibenwa**
These 2 years old, 3 release old migrations are making https://github.com/linagora/james-project/pull/3061 more complex hence with @mbaechler we proposed to drop them.

### CODE REVIEW

File: upgrade-instructions.md [hunk 1]
```diff
@@ -33,6 +33,17 @@ Change list:
...
+
+### Drop Cassandra schema version prior version 5
+
+Date 06/04/2020
+
+JIRA: https://issues.apache.org/jira/browse/JAMES-2997
+
+In an effort to simplify the code base, we dropped support for Cassandra schema version prior version 5.
+
+Installation running older schema version needs to rely on release 3.5.0 to upgrade to schema version 7.
```
Comment (rouazana) [inline upgrade-instructions.md:46]: maybe you can add that if you want to upgrade an older version you need to go through this and that version, no?

**Comment by chibenwa** [comment]
```

[9d8a901672962f814e0b50c3d0623f8c2dbb5495] [ERROR]   SolveMailboxInconsistenciesServiceTest.fixMailboxInconsistenciesShouldFailWhenVersionIsMissing:100 
[9d8a901672962f814e0b50c3d0623f8c2dbb5495] Expecting message:
[9d8a901672962f814e0b50c3d0623f8c2dbb5495]  <"Schema version 6 is required in order to ensure mailboxPathV2DAO to be correctly populated, got 2">
[9d8a901672962f814e0b50c3d0623f8c2dbb5495] but was:
[9d8a901672962f814e0b50c3d0623f8c2dbb5495]  <"Schema version 6 is required in order to ensure mailboxPathV2DAO to be correctly populated, got 5">
```

File: upgrade-instructions.md (hunk 1 above)
Comment (chibenwa) [reply upgrade-instructions.md:46]: Isn't it what I said here?

Comment (rouazana) [reply upgrade-instructions.md:46]: oh yes, in fact it's explained.
As it was a little hard for me, let's rewrite it:
```suggestion
Installation running older schema version than version 5 needs to rely on release 3.5.0 to upgrade to schema version 7, before upgrading to an eventual newer version.
```

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**